src/schnorrsig.rs

```diff
@@ -1,784 +0,0 @@
-//! # schnorrsig
```

Comment:
Is this removed because we do not use this?
Does it make it harder to keep this fork sync'd with upstream if we remove this? Is it worth considering leaving it in for ease of keeping the repo up to date?